test/e2e/basic_test.go

```diff
@@ -867,27 +866,10 @@ func testGratuitousARP(t *testing.T, data *TestData, namespace string) {
 // testClusterIdentity verifies that the antrea-cluster-identity ConfigMap is
 // populated correctly by the Antrea Controller.
 func testClusterIdentity(t *testing.T, data *TestData) {
-	clusterIdentityProvider := clusteridentity.NewClusterIdentityProvider(
-		antreaNamespace,
-		clusteridentity.DefaultClusterIdentityConfigMapName,
-		data.clientset,
-	)
-
-	const retryInterval = time.Second
-	const timeout = 10 * time.Second
-	var clusterUUID uuid.UUID
-	err := wait.PollUntilContextTimeout(context.Background(), retryInterval, timeout, true, func(ctx context.Context) (bool, error) {
-		clusterIdentity, _, err := clusterIdentityProvider.Get()
-		if err != nil {
-			return false, nil
-		}
-		clusterUUID = clusterIdentity.UUID
-		t.Logf("Cluster UUID: %v", clusterUUID)
-		return true, nil
-	})
-
-	assert.NoError(t, err, "Failed to retrieve cluster identity information within %v", timeout)
+	clusterUUID, err := data.getAntreaClusterUUID()
+	require.NoError(t, err, "Failed to retrieve cluster identity information within %v", timeout)
```

Comment:
It looks a bit confusing to log the global timeout while `data.getAntreaClusterUUID()` uses a local timeout, even though they have the same value. Passing the timeout as a parameter might look better?

Reply:
Thanks, I missed it when updating the code. Addressed.